### PR TITLE
Fix some deprecations and factor out a helper function

### PR DESCRIPTION
**Pull request by FatherInire**
Fixed some deprecation warnings related to string escaping and factored out a common function.

### CODE REVIEW

File: files/__main__.py [hunk 1]
```diff
@@ -19,12 +20,17 @@
...
+
 app = Flask(__name__, template_folder='templates')
 app.url_map.strict_slashes = False
 app.jinja_env.cache = {}
 app.jinja_env.auto_reload = True
 faulthandler.enable()
 
+if environ.get("SITE_ID") is None:
+	from dotenv import load_dotenv
+	load_dotenv(dotenv_path=Path("env"))
```
Comment (FatherInire) [inline files/__main__.py:32]: helps to run the e2e test locally

File: files/helpers/strings.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+# clean strings for searching
+def clean_string(my_str):
+	return my_str.replace(r'\\', '').replace('_', r'\_').replace('%', '').strip()
```
Comment (FatherInire) [inline files/helpers/strings.py]: not sure exactly what this is for but it was copy pasted a lot and seems to be related to querying the DB

Comment (zorbathut) [reply files/helpers/strings.py]: Snakes says this is for `ilike`-match sanitization. Rename this to something like `sql_ilike_clean()`?

Besides that, good catch and thanks for the cleanup here :D

Comment (FatherInire) [reply files/helpers/strings.py]: cool, done

**Review by zorbathut — APPROVED** [review]

**Pull request merged**